templates/tfengine/components/iam_members/variables.tf

```diff
@@ -0,0 +1,49 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+variable "storage_bucket_iam_members" {
```

Comment:
Why are all these variables hardcoded and not automatically generated as in [terraform.tfvars](https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/pull/955/files#diff-8e7b2e0f0e87b09d8fa06b91d7b9de9dca01cd0943dc2b06db64ef8be9c5fc27R15)?

If they're constant, I think we should simplify the `terraform.tfvars`, and if not, we should apply that range here too.

Reply:
because the description for different variables is slightly different. For example, for storage buckets, the description mentions that resource_ids should be a `list of storage bucket names`. While for other iam members it should be `a list of IDs` and not names.

Reply:
Ahh, I see. I think the difference is smaller but important enough to keep it, what a pain.

Anyway, the description is at the generated terraform config, and I think the user will input the variables before it's generated, so I see it more important to have the description of the fields (whether it is `name` or `ID`) at the schema, and that potentially will fix the issue.

As I said in https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/pull/955#discussion_r671477890, the discussion will be eventually held when applying `__schema__`

Not a big deal but the idea is to keep a single source of truth instead of repeating descriptions across the project. In this case, it's okay since it won't be repeated because the schema doesn't specify it.

Reply:
Alright, resolving this for now.

Reply:
alright, resolving this for now